db/seeds.rb

```diff
@@ -19,10 +19,10 @@
 users = User.all
 
 25.times do
-	Story.create!(
-		title:	Faker::Book.title,
-		body: 	Faker::Lorem.paragraph
-		)
+  Story.create!(
+    title:	Faker::Book.title,
+    body: 	Faker::Lorem.paragraphs(20).join('<p>').html_safe
```

Comment:
Since `paragraphs` returns an array and not a string of paragraphs, I join them with `<p>` to make each paragraph have spacing. The HTML safe tag lets it be translated directly as markup, so my fake paragraphs look legit!